lib/src/action.dart

```diff
@@ -20,11 +20,30 @@ import 'package:w_common/disposable.dart';
 
 import 'package:w_flux/src/constants.dart' show v3Deprecation;
 
+/// Like [Action1], but without an expected payload (aka a void payload).
+class Action0 extends Action1<void> {
```

Comment:
We may want to consider making this `extends Action<Null>` instead, to help prevent consumers from accidentally passing in values without knowing they'll be discarded.

Example: https://dartpad.dev/?id=97af2f0dd8c7919807bd6864ab085794

The relevant pieces from that ^
```dart
main() {
  {
    final action = VoidAction0();
    action();     
    action(null);
    action('foo');
    //     ^^^^^
    // No error or other indication that there's a potential issue with 
    // this usage.
    //
    // The value being passed in is discarded, but the consumer may not know that.
  }

  {
    final action = NullAction0();
    action();
    action(null);
    action('foo'); 
    //      ^^^^^
    // Error: The argument type 'String' can't be assigned to the parameter type 'Null'.
  }
}

class VoidAction0 extends Action1<void> {
  @override
  Future call([void payload]) => super.call(null);
}

class NullAction0 extends Action1<Null> {
  @override
  Future call([Null payload]) => super.call(null);
}
```